syscall/zsyscall_windows.go

```diff
@@ -18,7 +18,7 @@ var (
 )
 
 func GetModuleFileName(hmodule uint32, buf *uint16, buflen uint32) (n uint32, err error) {
-	r0, _, e1 := syscall.Syscall(procGetModuleFileNameW.Addr(), 3, uintptr(hmodule), uintptr(unsafe.Pointer(buf)), uintptr(buflen))
+	r0, _, e1 := syscall.Syscall(procGetModuleFileNameW.Addr(), 3, uintptr(hmodule), uintptr(unsafe.Pointer(buf)), uintptr(buflen)) // #nosec
```

Comment:
This is a generated file. It could be clobbered at any time by a future `go generate` command. See the file syscall_windows.go in this same directory. Is there another way of getting gas to ignore this file that doesn't involve modifying it? Or I guess we could always just redo the edit when necessary (since presumably gas will flag it as an error again).